Compression/src/CompressDetStepMCs_module.cc

```diff
@@ -0,0 +1,638 @@
+//
+// Compression module that takes detector steps (StrawGasSteps, CaloShowerSteps, and CrvSteps)
+// and compresses out unwanted MC information (StepPointMCs, SimParticles and GenParticles).
+// Note that the number of detector steps is NOT reduced.
+//
+// Allowed compression levels for different data products:
+// - StrawGasSteps : noCompression
+// - CaloShowerSteps : noCompression
+// - CrvSteps : noCompression
+// - SimParticles : noCompression, fullCompression
+// - StepPointMCs : noCompression, simParticleCompression
+// - MCTrajectories : noCompression, simParticleCompression
+//
```

Comment:
You need to add MCPrimary here too.  That (generally) gets produced in the same job as runs the step compression.

Reply:
Do you mean PrimaryParticle? I did have this in but removed it because there was a comment in the JobConfig scripts that said the FindMCPrimary module would need to be reconfigured for after compression. So I did that instead.